resource/dku-helpers.js

```diff
@@ -0,0 +1,93 @@
+/*
+Helper function to query webapp backend with a default implementation for error handling
+v 1.0.1
+*/
+
+function checkMandatoryParameters(param, webAppConfig) {
+    if (param.mandatory) {
+        var val = webAppConfig[param.name];
+        if (val == undefined || val == "") {
```

Comment:
```suggestion
        if (val === null || val === "") {
```
null returns true for both undefined and null, which is what you want unless you want null to be considered as ok.
You should also use `===` syntax that will run a deep check (verify type and value). For example, 1 == '1' returns true while 1 === '1' returns false. Using `===` makes you totally control what you want to check.

Reply:
Noted